app/components/NoAuthoritiesMessage.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
This this is only being used in one place, what do you think about colocating the file with the PartnersEdit screen?

Reply:
Two places -- Export Flow & Partners Edit